### PR TITLE
Remove patterns-base-fips from SLFO base containers

### PR DESCRIPTION
**Pull request by dirkmueller**
This is pulling crypto-policies and we have base-fips now instead.

### CODE REVIEW

File: src/bci_build/package/basecontainers.py [hunk 1]
```diff
@@ -173,11 +173,14 @@ def _get_supported_until_fips(os_version: OsVersion) -> datetime.date:
...
-            if os_version == OsVersion.SP3
-            else ["crypto-policies-scripts"]
+        package_list=(
+            [*os_version.release_package_names, "coreutils"]
+            + (
+                ["fipscheck"]
+                if os_version == OsVersion.SP3
+                else ["crypto-policies-scripts"]
+            )
+            + (["patterns-base-fips"] if os_version.is_slfo else [])
```
Comment (dcermak) [inline src/bci_build/package/basecontainers.py:183]: Note that this has no effect atm, as this container is not being built for SLE 16

**Review by msmeissn — COMMENTED** [review]
okay

File: src/bci_build/package/basecontainers.py [hunk 2]
```diff
@@ -144,7 +144,13 @@ def _get_fips_pretty_name(os_version: OsVersion) -> str:
...
+        case (
+            OsVersion.SP4
+            | OsVersion.SP5
+            | OsVersion.SP6
+            | OsVersion.SP7
+            | OsVersion.SLE16_0
+        ):
             return f"{os_version.pretty_os_version_no_dash} FIPS-140-3"
         case _:
             raise NotImplementedError(f"Unsupported os_version: {os_version}")
```
Comment (dcermak) [inline src/bci_build/package/basecontainers.py:156]: Wouldn't this be a bit simpler:
```Python
def _get_fips_pretty_name(os_version: OsVersion) -> str:
    if os_version == OsVersion.SP3:
        return f"{os_version.pretty_os_version_no_dash} FIPS-140-2"
    if os_version.is_sle15 or os_version.is_slfo:
        return f"{os_version.pretty_os_version_no_dash} FIPS-140-3"

    raise NotImplementedError(f"Unsupported os_version: {os_version}")
```

Comment (dirkmueller) [reply src/bci_build/package/basecontainers.py:156]: yes, we can do that.

**Review by dcermak — APPROVED** [review]

**Pull request merged**